style.css

```diff
@@ -0,0 +1,258 @@
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Montserrat:wght@100;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Crete+Round&family=Montserrat:wght@100;500&display=swap");
+
+* {
+  margin: 0;
+  border: 1px solid black;
+}
```

Comment:
- [Optional ] You did great work however all your elements have a solid black border. Kindly remove it and your project will look great just like the Figma design.